oca_port/cli/main.py

```diff
@@ -85,53 +102,58 @@
 @click.option("--no-cache", is_flag=True, help="Disable user's cache.")
 @click.option("--clear-cache", is_flag=True, help="Clear the user's cache.")
 def main(
-    from_branch: str,
-    to_branch: str,
     addon: str,
-    from_org: str,
-    from_remote: str,
+    source: str,
+    target: str,
+    destination: str,
+    source_version: str,
+    target_version: str,
     repo_name: str,
-    fork: str,
-    user_org: str,
+    upstream_org: str,
     verbose: bool,
     non_interactive: bool,
     output: str,
     fetch: bool,
     no_cache: bool,
     clear_cache: bool,
+    dry_run: bool,
 ):
-    """Migrate ADDON from FROM_BRANCH to TO_BRANCH or list Pull Requests to port
-        if ADDON already exists on TO_BRANCH.
+    """Migrate ADDON from SOURCE to TARGET or list Pull Requests to port
+        if ADDON already exists on TARGET.
+
+        E.g.:
+
+        $ oca-port OCA/server-tools#14.0 OCA/server-tools#16.0 auditlog
```

Comment:
```suggestion
    """Migrate ``addon``` from ``source`` to ``target`` or list PRs.
    
     If ``addon`` already exists....

        E.g.:

        $ oca-port OCA/server-tools#14.0 OCA/server-tools#16.0 auditlog
```